src/components/Notifier/Notice.tsx

```diff
@@ -0,0 +1,114 @@
+import { computed, defineComponent, onMounted, watch, onUnmounted } from 'vue';
+import type { CSSProperties } from 'vue';
+import classNames, { Argument } from 'classnames';
+import { NoticeProps } from './Notice.types';
+
+export default defineComponent<NoticeProps>({
+  name: 'Notice',
```

Comment:
Component name "Notice" should always be multi-word. => Cor-Notice maybe?

Reply:
i named it v-notice and also v-notifier good for you ?

Reply:
Cor for cornellius

Reply:
ok I renamed to 'cor-notice' and 'cor'-notifier'